docs/supply-chain-tools/reseller/reseller-enablement-guide.md

```diff
@@ -103,7 +103,7 @@ Table 1. Linux* OS-based Reseller Toolkit
 |    Connectivity        |    An Ethernet network to connect to the   IoT device.                                                                                                                                                                                                                                                                                                                            |
 |    Secure Key Fobs     |    YubiKey* security key (see <https://www.yubico.com/products/yubikey-hardware/>)                                                                                                                                                                                                                                                                                                  |
 |    Operating System    |    Ubuntu* OS version 16.x or 18.x                                                                                                                                                                                                                                                                                                                                                |
-|    Software            |    · Java\* Runtime Environment (JRE) software version 11 Oracle\* software (<http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html>) </p>  or  </p> **Open JRE** (<http://openjdk.java.net/install/index.html>) </p>  · Apache Tomcat\* (<http://tomcat.apache.org/>)   software version 8 or later </p>  · MariaDB\* (<http://mariadb.org/>) server version 10.4 or   later    |
+|    Software            |    · Java\* Runtime Environment (JRE) software version 11 Oracle\* software (https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html) </p>  or  </p> **Open JRE** (<http://openjdk.java.net/install/index.html>) </p>  · Apache Tomcat\* (<http://tomcat.apache.org/>)   software version 8 or later </p>  · MariaDB\* (<http://mariadb.org/>) server version 10.4 or   later    |
```

Comment:
We should be using reference to only OpenJDK here. You may remove Oracle JRE link.

Reply:
Done